jina/helper.py

```diff
@@ -714,9 +729,18 @@ def run(self):
     if loop and loop.is_running():
         # eventloop already exist
         # running inside Jupyter
-        thread = RunThread()
-        thread.start()
-        thread.join()
-        return thread.result
+        if is_jupyter():
+            thread = RunThread()
+            thread.start()
+            thread.join()
+            try:
+                return thread.result
+            except AttributeError:
+                from .excepts import BadClient
+                raise BadClient('something wrong when running the eventloop, result can not be retrieved')
+        else:
+            raise RuntimeError('you have an eventloop running but not using jupyter/ipython, '
```

Comment:
`jinad` runs inside `fastapi+uvicorn` which has a default event loop (again based on uvloop). Why do we want to raise a `RuntimeError`?